src/pfs/pfs.proto

```diff
@@ -631,6 +644,8 @@ service API {
   rpc SquashCommitSet(SquashCommitSetRequest) returns (google.protobuf.Empty) {}
   // DropCommitSet drops the commits of a CommitSet and all data included in the commits.
   rpc DropCommitSet(DropCommitSetRequest) returns (google.protobuf.Empty) {}
+  // SearchForFileInBranch searches for commits that reference a supplied file being modified in a branch.
+  rpc SearchForFileInBranch(SearchForFileInBranchRequest) returns (SearchForFileInBranchResponse) {}
```

Comment:
The Unary rpc makes the protocol simpler (since we only need one message type).  But the other methods that list commits return streams.  We should probably be consistent with the rest of the API, unless there is a plan for migration to unary RPCs for iteration.

I don't think I'm up to date on the latest thinking there, so I'd defer to @armaanv or someone else on CORE

Reply:
Separately, this is a descriptive name, but it might be worth making this more general.

In the design doc you had proposed `FindCommits`, which searches for commits matching arbitrary criteria.  I don't think we should deviate too far from that model/level of generality.

Reply:
Gotcha, `FindCommits` is probably a good candidate for being general enough while retaining intention/description

Reply:
Also, we did agree on using streams which is something I forgot about while doing the implementation, so that should be easy enough to swap to